src/go/rpk/pkg/cli/security/role/describe.go

```diff
@@ -168,7 +172,7 @@ func describeAndPrintRole(ctx context.Context, admCl *adminapi.AdminAPI, kafkaAd
 }
 
 func describedToRoleAcl(results kadm.DescribeACLsResults) []roleACl {
-	var ret []roleACl
+	ret := []roleACl{}
```

Comment:
I actually like the var definition more but, eh it's the same anyway

Reply:
It's on purpose. I also like the other way, but I changed it because I don't want `ret` to be nil if it's empty, mainly to avoid showing:
```
{permissions:null}
```
After going through the JSON marshaller